components/slider.css

```diff
@@ -0,0 +1,18 @@
+.slider-container {
+  width: 100%;
```

Comment:
In place of % you can use rem for bigger widths and heights but for smaller units percentage is good enough . The code quality looks good and it is very well indented. Just  make the percentages in rems.